cis/schema.json

```diff
@@ -29,54 +29,54 @@
     "properties": {
         "user_id": {
             "type": "string",
-            "description": "(Authoritative) The user ID as defined by the underlying IdP. Profile consumers should not try to interpret this value in any way, but may use it as a stable identifier for this user as identified by this IDP."
+            "description": "The unique user identifier. Profile consumers should not try to interpret this value in any way."
```

Comment:
i don't want readers to infer that this is to be used as the user identifier for users to login, i.e.
Login: blah|LDAP|xxx
Password: xxxx

Because the user_id is not very human friendly compared to emails. It's ok to use it internally where needed though.